src/Data/VerifiedEq/Instances/Generics.hs

```diff
@@ -1,78 +1,84 @@
-{-@ LIQUID "--higherorder"        @-}
-{-@ LIQUID "--exactdc"            @-}
+{-@ LIQUID "--reflection" @-}
+{-@ LIQUID "--ple"        @-}
+
 {-# LANGUAGE EmptyCase            #-}
 
 module Data.VerifiedEq.Instances.Generics where
 
-import GHC.Generics
+import GHC.Generics hiding (unPar1)
 import Data.VerifiedEq
 import Language.Haskell.Liquid.ProofCombinators
 
-{-@ data V1 @-}
 
-{-@ axiomatize eqV1 @-}
+{-@ reflect eqV1 @-}
 eqV1 :: V1 p -> V1 p -> Bool
 eqV1 _ _ = False
 -- eqV1 x _ = case x of {}
 
 absurd :: V1 p -> a
 absurd x = case x of {}
 
-{-@ eqV1Refl :: x:V1 p -> { eqV1 x x } @-}
+{-@ eqV1Refl :: x:_ -> { eqV1 x x } @-}
```

Comment:
Is it considered idiomatic LH nowadays style to supply partial type signatures like this? (Honest question—I'm not sure if there's a right answer here or not.)

Reply:
Yes, both aesthetically -- the other stuff seems redundant, but more importantly, in this case, there's some weirdness with the actual type so `V1 p` was inconsistent with the GHC type. 
(Can't recall what was going on ... but LH gives the suggestion 

```
when this happens, please use '_'
```

and that did the trick here :-)